openedx/features/course_experience/views/course_home.py

```diff
@@ -130,7 +126,7 @@ def render_to_fragment(self, request, course_id=None, **kwargs):
             'course_sock_fragment': course_sock_fragment,
             'disable_courseware_js': True,
             'uses_pattern_library': True,
-            'show_reviews_link': show_reviews_link,
+            'request': request,
```

Comment:
Nit: I'd list this as the first parameter since they go from most generic to most specific.